src/vm/arm64/PInvokeStubs.asm

```diff
@@ -64,8 +64,8 @@ __PInvokeGenStubFuncName SETS "$__PInvokeGenStubFuncName":CC:"_RetBuffArg"
             ; The way we do this is to shift the managed target to the left by one bit and then set the
             ; least significant bit to 1.  This works because MethodDesc* are always 8-byte aligned.
             ;
-            lsl             x9, x9, #1
-            orr             x9, x9, #1
+            lsl             x12, x12, #1
+            orr             x12, x12, #1
```

Comment:
I think this should be
```
lsl             $HiddenArg, $HiddenArg, #1
orr            $HiddenArg, $HiddenArg, #1
```

Reply:
@sdmaclea Yea, that's better. Will change it.